docs/images/after/output-newline.svg

```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="200" height="100">
```

Comment:
Does "before" image is missing?

Reply:
I'm re-using another example's after image as the before image, but if desired I can copy it.